Adapter/MemcachedCache.php

```diff
@@ -11,8 +11,16 @@
 
 namespace Sonata\CacheBundle\Adapter;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\MemcachedCache class is deprecated since version 2.4 and will be removed in 3.0.'
```

Comment:
I see. Don't change it, to much work, but next time, pick the class solution please. Shorter and less error prone